uint8 taskType
bool taskRequiresPoi
geometry_msgs/Point taskPoi
---
uint8 result   # 0=ACCEPTED, 1=TAKE_ME_THERE, 2=FINISHED, 3=BUSY
